package.json

```diff
@@ -43,12 +43,16 @@
   "devDependencies": {
     "@ethersproject/address": "^5.3.0",
     "@ethersproject/contracts": "^5.3.0",
+    "@ethersproject/experimental": "^5.4.0",
     "@ethersproject/networks": "^5.3.0",
     "@ethersproject/providers": "^5.3.0",
     "@ethersproject/solidity": "^5.3.0",
     "@types/big.js": "^4.0.5",
     "@types/jest": "^24.0.25",
+    "axios": "^0.21.1",
```

Comment:
Great work Sarang. Only thing worth mentioning is that it would be good if we could replace axios with isomorphic-unfetch to not include additional http libraries.

Reply:
Sure, I updated axios to isomorphic-unfetch.